packages/core/src/test/embedded.test.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { isEmbedded } from '../shared/extensionGlobals'
+
+describe('isEmbedded', function () {
```

Comment:
I assume more tests for "embedded mode" will be added here. Otherwise it might make more sense to create a `globals.test.ts` or `startup.test.ts` for testing various init state.

Reply:
OK. If I add one of these files and include only tests relevant to the `embedded` global, will that be sufficient?

Reply:
yes